android/settings.gradle

```diff
@@ -1,5 +1,7 @@
 rootProject.name = 'Mattermost'
 include ':app'
+include ':detox'
+project(':detox').projectDir = new File(rootProject.projectDir, '../detox/node_modules/detox/android')
```

Comment:
we dont need this to build for production apps. Think if we can inject via script